ui/imports/shared/views/PasswordView.qml

```diff
@@ -222,6 +222,10 @@ Column {
 
                 // Update strength indicator:
                 strengthInditactor.strength = d.convertStrength(RootStore.getPasswordStrengthScore(newPswInput.text, root.onboarding))
+
+                if(textField.text.length === confirmPswInput.text.length || textField.text.length === 0) {
```

Comment:
Do we really want to check whether the password matches when the `textField.text.length === 0`?
At that point we already know there's practically no password at all

Reply:
I'm not sure I understand the purpose for this check `if(textField.text.length === confirmPswInput.text.length || textField.text.length === 0)`.
It looks to me that both checks are done in `checkPasswordMatches`, and this "optimization" doesn't make any improvement or do I miss something?

Reply:
Guys I don't know actually, I just moved some checks from other component. But I will check and fix if it's possible.

Reply:
I fixed it